pkg/ddevapp/ddevapp.go

```diff
@@ -1097,6 +1097,59 @@ func (app *DdevApp) ComposeFiles() ([]string, error) {
 	return orderedFiles, nil
 }
 
+// EnvFiles returns a list of env files for a project.
+// It has to put the .ddev/.env first
+// It has to put the .ddev/.env.* second
+// It has to put the .ddev/.env.*.override third
+// It has to put the .ddev/.env.override last
+// Env files ending with .example are ignored.
```

Comment:
Maybe I didn't fully describe this behavior in the docs, probably it could be better.